crates/sui-adapter/src/programmable_transactions/execution.rs

```diff
@@ -271,6 +274,14 @@ fn execute_command<E: fmt::Debug, S: StorageView<E>, Mode: ExecutionMode>(
         Command::Publish(modules) => {
             execute_move_publish::<_, _, Mode>(context, &mut argument_updates, modules)?
         }
+        Command::Upgrade(upgrade_ticket, dep_ids, modules) => {
+            vec![execute_move_upgrade(
```

Comment:
nit, make `execute_move_upgrade` return a `Vec`, just to be a bit more inline/idiomatic